radio/src/targets/taranis/CMakeLists.txt

```diff
@@ -625,7 +625,7 @@ if(PCB STREQUAL XLITE OR PCB STREQUAL XLITES OR PCBREV STREQUAL COMMANDO8)
     targets/common/arm/stm32/sticks_pwm_driver.cpp)
 endif()
 
-if(PCBREV STREQUAL MT12 OR PCBREV STREQUAL T14)
+if(PCBREV STREQUAL MT12)
```

Comment:
Maybe in future these should be more feature flags rather than radio specific? i.e. I could see this and the above (sticks_pwm_driver) being more generic...